torch/testing/_internal/common_utils.py

```diff
@@ -1443,3 +1445,16 @@ def load_tests(loader, tests, pattern):
             check_test_defined_in_running_script(test)
             test_suite.addTest(test)
     return test_suite
+
+
+def _assertGradAndGradgradChecks(test_case, apply_fn, inputs):
+    # call assert function rather than returning a bool since it's nicer
+    # if we get whether this failed on the gradcheck or the gradgradcheck.
+    test_case.assertTrue(gradcheck(apply_fn, inputs))
+    test_case.assertTrue(gradgradcheck(apply_fn, inputs))
+
+
+dtype2prec = {torch.float: 1e-5,
```

Comment:
we _really_ shouldn't be canonicalizing this.

1) The name is really bad (it's _maybe_ good enough for some one-off tests in test_nn, but not as a common utility)
2) The numbers were selected empirically for some backwards cases, there's nothing special about them.

So we don't have to restructure the tests, can we rename this something like "dtype2prec_DONTUSE" and write a comment here about the recommended way of doing this (use `@precisionOverride` or just find a reasonable number for your use case).